### DISCUSSION TITLE
brew install matplotplusplus

### DISCUSSION THREAD

**Discussion by ankane** [alandefreitas/matplotplusplus#79] [Show and tell] [answered]
Hi, thanks for this neat project! Just wanted to let you know it's now available on Homebrew, so Mac users can do:

```sh
brew install matplotplusplus
```

Ref: https://github.com/Homebrew/homebrew-core/pull/62577

**Comment by alandefreitas** [comment, marked as answer]
Hi @ankane,

That's very good news! 😀 I updated the docs to mention it.
Let me know if there's anything we should do to make the library more easily packageable.

Also welcome as a contributor "honoris causa": 😀
https://github.com/alandefreitas/matplotplusplus#contributors
https://alandefreitas.github.io/matplotplusplus/contributing/contributors.html

**Reply by ankane** [reply]
Thanks @alandefreitas. Installation was pretty easy / standard, so 👍
